packages/protocol/test-sol/devchain/migration/Migration.t.sol

```diff
@@ -215,8 +210,8 @@ contract EpochManagerIntegrationTest is IntegrationTest, MigrationsConstants {
 
   function test_Reverts_WhenEndOfEpochHasNotBeenReached() public {
     // fund treasury
-    vm.prank(address(0));
-    celoTokenContract.mint(unreleasedTreasury, L2_INITIAL_STASH_BALANCE);
+    // vm.prank(address(0));
```

Comment:
If this code is no longer necessary, remove entirely rather than commenting out.